static/src/js/app/components/quick-add-button/QuickAddVocabForm.vue

```diff
@@ -180,29 +176,30 @@
           this.submitting = false;
           return;
         }
-        if (this.latticeNodeId !== null) {
-          const node = this.latticeNode.find(
-            (ele) => parseInt(ele.pk, 10) === parseInt(this.latticeNodeId, 10),
-          );
-          if (node.definition !== this.definition) {
-            this.latticeNodeId = null;
-          }
-        }
 
-        await this.$store.dispatch(CREATE_PERSONAL_VOCAB_ENTRY, {
+        const newEntryData = {
           headword,
           definition,
           vocabularyListId: vocabularyListItem.id,
           familiarity: this.familiarityRating,
           lang: vocabularyListItem.lang,
-        });
+          lemmaId: null,
+        };
+        if (this.lemmaId) {
+          newEntryData.lemmaId = this.lemmaId;
+        }
+        console.log(newEntryData);
```

Comment:
Think we should remove the console logs.

Reply:
good catch, thanks!